config/Makefile

```diff
@@ -18,5 +18,5 @@ SUFFIXES:
 include ../Makefile.INCLUDE
 
 generated/config.pb.go: config.proto
-	$(GO_GET) code.google.com/p/goprotobuf/protoc-gen-go
+	$(GO) get github.com/golang/protobuf/protoc-gen-go
```

Comment:
Sorry, yes. Now it says a similar thing for git:

```
package github.com/golang/protobuf/protoc-gen-go: /home/julius/gosrc/src/github.com/prometheus/prometheus/.build/root/gopath/src/github.com/golang/protobuf exists but /home/julius/gosrc/src/github.com/prometheus/prometheus/.build/root/gopath/src/github.com/golang/protobuf/.git does not - stale checkout?
```

Actually, this still happens with `$(GO)` (not sure why it seemingly didn't when I tested it) because `$(GO)` already includes the custom `GOPATH` to which everything has been copied. So either we just make this literally `go get` (depending on someone's local Go installation - it already depends on a locally installed protoc anyways), or we already check in the `protoc-gen-go` package as well. But Godeps is quite stubborn in what it will automatically save to the vendor directory (it really looks at your source code and stores exactly all transitive dependencies, and nothing else). Maybe Godep is the wrong tool after all and we should just manually vendor with less hassle... Hmm.